modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +88,34 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+
+[discrete]
+[id="configuring-csm-for-fujitsu-irmc_{context}"]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
+
+The Compatibility Support Module (CSM) configuration provides support for legacy BIOS backward compatibility with UEFI systems. You must configure the CSM when you deploy a cluster with Fujitsu iRMC, otherwise the installation might fail.
+
+[IMPORTANT]
+====
+Before you configure the CSM, ensure that you have disabled *Secure Boot Control*. You can disable *Secure Boot control* under *Security* -> *Secure Boot Configuration*.
+====
+
+[NOTE]
+====
+To configure the CSM, refer to the hardware guide for the node.
+====
+
+.Procedure
+. Boot the node and select the BIOS menu.
+. Under *Advanced* tab, select *CSM Configuration* from the list.
+. Enable the *Launch CSM* and set the following values:
++
+[options="header"]
+|===
+|Item |Value
+| *Boot option filter* | UEFI and Legacy
+| *Launch PXE OpROM Policy* | UEFI only
```

Comment:
I noticed that "Policy" is in uppercase in this line, but in lower case in the line that follows. You could update these items to match the case used in the UI element.

Reply:
Ah, these items are currently consistent with the UI element.
![Screenshot from 2022-05-26 14-51-57](https://user-images.githubusercontent.com/53652350/172119994-d957bc22-d225-4d4b-96ca-182466c86be4.png)

Reply:
Thank you for clarifying that.